modules/v2x/proto/v2x_obu_traffic_light.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto2";
+
+package apollo.obu.v2x;
+
+import "modules/common/proto/header.proto";
+
+message NextTrafficLightInfo {
+  enum Color {
+    UNKNOWN = 0;
+    RED = 1;
+    YELLOW = 2;
+    GREEN = 3;
+    BLACK = 4;
+    FLASH_GREEN = 5;
+  };
+  optional Color color = 1;
+  optional int32 start_time = 2;
+  optional int32 end_time = 3;
+};
+message SingleTrafficLight {
+  enum Color {
+    UNKNOWN = 0;
+    RED = 1;
+    YELLOW = 2;
+    GREEN = 3;
+    BLACK = 4;
+    FLASH_GREEN = 5;
+  };
+  enum Type {
+    STRAIGHT = 1;
+    LEFT = 2;
+    RIGHT = 3;
+    U_TURN = 4;
+  };
+
+  optional Color color = 1;
+  optional int32 trafficlight_type = 2;
+  // Traffic light string-ID in the map data.
+  optional string id = 3;
+  optional int32 color_remaining_time_s = 4;
+  optional bool right_turn_light = 5;
+  // v2x next traffic light color
+  optional Color next_color = 6;
+  // v2x next traffic light remaining time
+  optional double next_remaining_time = 7;
+  // next 2nd traffic ligth color
+  optional Color next_2nd_color = 8;
+  optional double next_2nd_remaining_time = 9;
+}
+
+message LaneTrafficLight {
+  // Feature points of lane in the map reference frame.
+  optional double gps_x_m = 1;
+  optional double gps_y_m = 2;
+  repeated SingleTrafficLight single_traffic_light = 3;
+}
+
+message RoadTrafficLight {
+  repeated LaneTrafficLight lane_trafficlight = 1;
+  optional int32 road_direction = 2;
+}
+message V2X_IntersectionTrafficLightData {
```

Comment:
should the `V2X_` prefix removed. And this file should be renamed to `obu_traffic_light`.